addons/survey/models/survey_survey.py

```diff
@@ -230,7 +233,7 @@ def _check_answer_creation(self, user, partner, email, test_entry=False):
                     raise UserError(_('Creating token for external people is not allowed for surveys requesting authentication.'))
             if self.access_mode == 'internal' and (not user or not user.has_group('base.group_user')):
                 raise UserError(_('Creating token for anybody else than employees is not allowed for internal surveys.'))
-            if not self.has_attempts_left(partner or (user and user.partner_id), email):
+            if not self.has_attempts_left(partner or (user and user.partner_id), email, invite_token):
```

Comment:
Check this only if `invite_token` is given. "retry" case.
Check attempt limit makes no sense when creating a new user_input.

Reply:
Not necessary anymore (as discussed).